acceptance/Gemfile

```diff
@@ -12,7 +12,7 @@ def location_for(place, fake_version = nil)
   end
 end
 
-gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 3.5.0')
+gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 3.5')
```

Comment:
Can we change this to `'~> 3.10'` to indicate we need a minimum Beaker version?  IMHO that more clearly conveys the need for at least a min version to support Ubuntu 16.10 (per the ticket).

It should also make anyone with an already installed gemset locally blow up if they try to `bundle exec beaker` after updating to this code, should they have an old Beaker version written into `Gemfile.lock` (which is a good thing I think)